pkg/reconciler/kubernetes/tektontrigger/transform.go

```diff
@@ -27,8 +27,10 @@ import (
 
 // Triggers ConfigMap
 const (
-	ConfigDefaults = "config-defaults-triggers"
-	FeatureFlag    = "feature-flags-triggers"
+	ConfigDefaults       = "config-defaults-triggers"
+	FeatureFlag          = "feature-flags-triggers"
+	DefaultRunAsUserKey  = "default-run-as-user"
```

Comment:
can we get the key from vendor if exported, otherwise fine

Reply:
Its not exposed

But will make a note thank you